api/client/samples/crop_models/Sao_Paulo_brazil_sugar_content_modeling.ipynb

```diff
@@ -0,0 +1,3564 @@
+{
```

Comment:
<p>Failed here with this error:</p><p> </p><p><strong>---------------------------------------------------------------------------</strong></p><p><strong>KeyError</strong>                                  Traceback (most recent call last)</p><p><strong>c:\users\justi\appdata\local\programs\python\python37\lib\site-packages\pandas\core\indexes\base.py</strong> in get_loc<strong>(self, key, method, tolerance)</strong></p><p>   2656             <strong>try:</strong></p><p><strong>-&gt; 2657                 return</strong> self<strong>.</strong>_engine<strong>.</strong>get_loc<strong>(</strong>key<strong>)</strong></p><p>   2658             <strong>except</strong> KeyError<strong>:</strong></p><p> </p><p><strong>pandas\_libs\index.pyx</strong> in pandas._libs.index.IndexEngine.get_loc<strong>()</strong></p><p> </p><p><strong>pandas\_libs\index.pyx</strong> in pandas._libs.index.IndexEngine.get_loc<strong>()</strong></p><p> </p><p><strong>pandas\_libs\hashtable_class_helper.pxi</strong> in pandas._libs.hashtable.PyObjectHashTable.get_item<strong>()</strong></p><p> </p><p><strong>pandas\_libs\hashtable_class_helper.pxi</strong> in pandas._libs.hashtable.PyObjectHashTable.get_item<strong>()</strong></p><p> </p><p><strong>KeyError</strong>: 'intercept'</p><p> </p><p>During handling of the above exception, another exception occurred:</p><p> </p><p><strong>KeyError</strong>                                  Traceback (most recent call last)</p><p><strong>&lt;ipython-input-36-9671c44818da&gt;</strong> in &lt;module&gt;</p><p>      1 <strong>for</strong> name <strong>in</strong> <strong>['lst','pet','trmm','ndvi','intercept',</strong> <strong>'ATR']:</strong></p><p><strong>----&gt; 2     </strong>df_tog<strong>[</strong>name<strong>]</strong> <strong>=</strong> df_tog<strong>[</strong>name<strong>].</strong>astype<strong>(</strong>float<strong>)</strong></p><p> </p><p><strong>c:\users\justi\appdata\local\programs\python\python37\lib\site-packages\pandas\core\frame.py</strong> in __getitem__<strong>(self, key)</strong></p><p>   2925             <strong>if</strong> self<strong>.</strong>columns<strong>.</strong>nlevels <strong>&gt;</strong> <strong>1:</strong></p><p>   2926                 <strong>return</strong> self<strong>.</strong>_getitem_multilevel<strong>(</strong>key<strong>)</strong></p><p><strong>-&gt; 2927             </strong>indexer <strong>=</strong> self<strong>.</strong>columns<strong>.</strong>get_loc<strong>(</strong>key<strong>)</strong></p><p>   2928             <strong>if</strong> is_integer<strong>(</strong>indexer<strong>):</strong></p><p>   2929                 indexer <strong>=</strong> <strong>[</strong>indexer<strong>]</strong></p><p> </p><p><strong>c:\users\justi\appdata\local\programs\python\python37\lib\site-packages\pandas\core\indexes\base.py</strong> in get_loc<strong>(self, key, method, tolerance)</strong></p><p>   2657                 <strong>return</strong> self<strong>.</strong>_engine<strong>.</strong>get_loc<strong>(</strong>key<strong>)</strong></p><p>   2658             <strong>except</strong> KeyError<strong>:</strong></p><p><strong>-&gt; 2659                 return</strong> self<strong>.</strong>_engine<strong>.</strong>get_loc<strong>(</strong>self<strong>.</strong>_maybe_cast_indexer<strong>(</strong>key<strong>))</strong></p><p>   2660         indexer <strong>=</strong> self<strong>.</strong>get_indexer<strong>([</strong>key<strong>],</strong> method<strong>=</strong>method<strong>,</strong> tolerance<strong>=</strong>tolerance<strong>)</strong></p><p>   2661         <strong>if</strong> indexer<strong>.</strong>ndim <strong>&gt;</strong> <strong>1</strong> <strong>or</strong> indexer<strong>.</strong>size <strong>&gt;</strong> <strong>1:</strong></p><p> </p><p><strong>pandas\_libs\index.pyx</strong> in pandas._libs.index.IndexEngine.get_loc<strong>()</strong></p><p> </p><p><strong>pandas\_libs\index.pyx</strong> in pandas._libs.index.IndexEngine.get_loc<strong>()</strong></p><p> </p><p><strong>pandas\_libs\hashtable_class_helper.pxi</strong> in pandas._libs.hashtable.PyObjectHashTable.get_item<strong>()</strong></p><p> </p><p><strong>pandas\_libs\hashtable_class_helper.pxi</strong> in pandas._libs.hashtable.PyObjectHashTable.get_item<strong>()</strong></p><p> </p><p><strong>KeyError</strong>: 'intercept'</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='63'/>

Reply:
<p>Fixed</p>